README.md

```diff
@@ -31,6 +31,34 @@ For running some specific tests you can do this as following:
 
     $ python -m unittest tests.test_publisher
 
+
+Installation
+------------
+
+Install `skt` directly from git:
+
+    $ pip install git+https://github.com/RH-FMK/skt
+
+The `skt` executable should work after installation:
+
+    $ skt -h
+    usage: skt [-h] [-d WORKDIR] [-w] [--junit JUNIT] [-v] [--rc RC] [--state]
+            {merge,build,publish,run,report,cleanup,all,bisect} ...
+
+    positional arguments:
+    {merge,build,publish,run,report,cleanup,all,bisect}
+
+    optional arguments:
+    -h, --help            show this help message and exit
+    -d WORKDIR, --workdir WORKDIR
+                            Path to work dir
+    -w, --wipe            Clean build (make mrproper before building), remove
+                            workdir when finished
+    --junit JUNIT         Path to dir to store junit results in
+    -v, --verbose         Increase verbosity level
+    --rc RC               Path to rc file
+    --state               Save/read state from 'state' section of rc file
+
```

Comment:
Could we perhaps not include the `skt -h` output here, and e.g. simply say that `skt -h` should produce command-line help, or such? This way README.md would be shorter and we won't have to update it every time we change the `skt -h` output.

Reply:
That works for me.